sdk/storage/storage-blob/ChangeLog.md

```diff
@@ -3,6 +3,10 @@
 ## 2019.10 12.0.0-preview.5
 
 - [Breaking] `IPRange` is renamed to `SasIPRange`.
+- Added ADLSgen2 related APIs.
+  - Added new `DirectoryClient` for directory related operations.
+  - Added `ContainerClient.getDirectoryClient()` to get a sub directory client under specific container.
+  - Added `BlobClient.move()`, `BlobClient.setPermissions()`, `BlobClient.setAccessControl()` and `BlobClient.getAccessControl()`.
```

Comment:
I understand the new `DirectoryClient` being introduced for ADLS, but can you point me to some docs or clarify how the `move()`, `setPermissions()`, `setAccessControl()` and `getAccessControl()` methods on the BlobClient are related ADLS?